src/main/java/beans/AuthenticatedRequestBean.java

```diff
@@ -0,0 +1,32 @@
+package beans;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+/**
+ * Created by benrudolph on 9/8/16.
+ */
+public class AuthenticatedRequestBean {
```

Comment:
Nice, really like this session vs. not breakdown. Might be good to add a comment clarifying when to use one vs the other